packages/curve-ui-kit/src/widgets/Footer/Description.tsx

```diff
@@ -39,7 +39,7 @@ export const Description = () => (
         color="textPrimary"
         sx={{
           fontSize: Sizing[600], // This is a one-off specifically for the logo text
```

Comment:
This does need adjustment now though. Because you can't easily apply `!important` to this and that's a dirty global hack anyway, you can use the self-referencing `&` operator to increase specificity:

```ts
'&': {
  fontSize: Sizing[600], // This is a one-off specifically for the logo text
},
```